### DISCUSSION TITLE
Clarity re Document Node Connector

### DISCUSSION THREAD

**Discussion by Farscaper** [FlowiseAI/Flowise#798] [Ideas]
Since it is clear that people routinely confuse standard Documents (Word docs, pdfs, etc.) with Chunked Documents it may help to rename the node connection point from Document to Chunked Document or some other differentiator.
https://share.getcloudapp.com/nOuLZpKd

**Comment by chungyau97** [comment]
Hi @Farscaper

Feel free to open the PR, it's not hard but there're a lot of files to be change and test again before release
Files to change in these folders [document loaders](https://github.com/FlowiseAI/Flowise/tree/main/packages/components/nodes/documentloaders) and [vector stores](https://github.com/FlowiseAI/Flowise/tree/main/packages/components/nodes/vectorstores)

Examples:
1. [Folder.ts line: 25](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/documentloaders/Folder/Folder.ts#L25) from `this.type = 'Document'` to `this.type = 'DocumentLoader'`
2. [Chroma_Upsert.ts line: 41-43](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/vectorstores/Chroma/Chroma_Upsert.ts#L41C16-L41C16) from
```
                label: 'Document',
                name: 'document',
                type: 'Document',
```
to         
```
                label: 'Document Loader',
                name: 'documentLoader',
                type: 'DocumentLoader',
```
3. [Chroma_Upsert.ts line: 88](https://github.com/FlowiseAI/Flowise/blob/main/packages/components/nodes/vectorstores/Chroma/Chroma_Upsert.ts#L88) from `const docs = nodeData.inputs?.document as Document[]` to `const docs = nodeData.inputs?.documentLoader as Document[]`
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/b93b18eb-037f-4348-ab23-fb5154036e5a)

cc @HenryHengZJ

**Reply by Farscaper** [reply]
Thanks for your consideration. That sounds like a great idea, except for a couple things.

First, I'd be bound to screw it up somehow.
Second, I would not want anyone to make changes based solely on my personal opinion. There should perhaps be a poll or something? 

Besides, maybe no one else thinks it's an issue. Now that it's clear to me what it does, it's less of one for me. I was thinking of new users. Looking at it now I'm not sure I want a change at all. Document looks fine even though I think people should understand that it's not their original source document that gets sent to the vector store.

If a change was needed I might prefer Chunk Loader or Chunked Document to Document Loader but maybe someone has something even better.  For now, Document is fine.

**Reply by HenryHengZJ** [reply]
yeah I think what @chungyau97 meant was you can change that on your forked repo. For now, we will stick to the name
